src/Learning/KDDomainKnowledge/KDMultiTableFeatureConstruction.cpp

```diff
@@ -1589,13 +1589,18 @@ void KDMultiTableFeatureConstruction::ComputeAllClassesCompliantRules(
 			 outputClassDomainCompliantRules->GetAllClassesCompliantRules()->GetAt(nClass));
 		kwcClass = classCompliantRules->GetClass();
 
-		// On interdit les cles de la classe
-		// De facon generale, il s'agit d'un principe: la cle ne sert qu'a encoder une structure et la
-		// memoriser: il ne s'agit pas d'attributs porteurs d'information. Pour la classe racine, les cle
-		// apparaissent une seule fois instance, et ne peuvent etre informatives. Pour les classes secondaires
-		// inclues, les cles sont soient unique par instance principale (la cle de l'incluant) sans interet,
-		// soit avec un role d'identifiant dans la table secondaire, sans interet autre que compter le nombre
-		// d'enregistrements (par CountDistinct, ici redondant avec Count).
+		// On interdit l'utilisation des cles du schema-multi-table pour la construction de variables.
+		// Les cles ne sont pas des attributs porteurs d'information.
+		// Pour la classe racine, les cles apparaissent une seule fois par instance, et ne peuvent pas etre informatives.
```

Comment:
Shouldn't we rather mention "classe principale" ("main class" / "main table") rather than "root"?

Reply:
Effectivement: ce sera corrigé
- cette PR a été fait indépendamment de la PR https://github.com/KhiopsML/khiops/pull/484 qui a pris en charge le renommage systématique selon root ou main
- et ici, il s'agit bien du cas "main"